onnx/defs/tensor/defs.cc

```diff
@@ -1897,14 +1888,8 @@ ONNX_OPERATOR_SET_SCHEMA(
           const auto& input_shape = ctx.getInputType(0)->tensor_type().shape();
           const auto input_ndim = input_shape.dim_size();
           const auto output_ndim = input_ndim + static_cast<int>(axes.size());
-          for (auto& axe : axes) {
-            if (axe < -output_ndim || axe >= output_ndim) {
-              fail_shape_inference("values in 'axes' are beyond the bounds of the computed output shape");
-            }
-            if (axe < 0) {
-              axe += output_ndim;
-            }
-          }
+          checkAxesRange(axes, output_ndim);
```

Comment:
could these lines above be replaced with checkDuplicateAxes()?
https://github.com/jantonguirao/onnx/blob/bbab6140eb8450205027400911457a0073c832a2/onnx/defs/tensor/defs.cc#L1878-L1885